_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -122,10 +122,61 @@ This `iframe` element contains a link that is not part of its [sequential focus
 <iframe tabindex="-1" srcdoc="<a href='/' tabindex='-1'>Home</a>"></iframe>
 ```
 
+#### Inapplicable Example 5
+
+This `iframe` element is [inert][] because of its own `inert` [attribute value][].
+
+```html
+<iframe inert srcdoc="<a href='/'>Home</a>"></iframe>
+```
+
+#### Inapplicable Example 6
+
+Once "Privacy policy details" button is activated, the `iframe` element become [inert][] because of showModal() method that causes the `iframe` to be [blocked by a modal].
+
+```html
+<iframe srcdoc="<a href='/'>Home</a>"></iframe>
+<div>
+    <button id="ppDetails">Privacy policy details</button>
+</div>
+<dialog id="ppDialog" aria-labelledby="dialogLabel">
+    <h2 id="dialogLabel">Privacy Policy</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <button id="cancel">Cancel</button>
+</dialog>
+<script>
+    const ppDetails = document.getElementById("ppDetails");
+    const cancelButton = document.getElementById("cancel");
+    const dialog = document.getElementById("ppDialog");
+
+    ppDetails.addEventListener("click", () => {
+        dialog.showModal();
+    });
+
+    cancelButton.addEventListener("click", () => {
+        dialog.close();
+    });
+</script>
+```
+
+#### Inapplicable Example 7
+
+This `iframe` element is [inert][] because it is [blocked by a modal].
```

Comment:
It's technically not true. The HTML definition of "blocked by a modal" requires a `dialog` element to do the blocking, while this is done by a `div`.

Also, this actually doesn't works. Due to non-interference with the host language, `aria-modal` does not make the rest of the page inert. [ARIA for `aria-modal`](https://www.w3.org/TR/wai-aria-1.2/#aria-modal) states:
> When a modal element is displayed, authors SHOULD mark all other contents as inert (such as "inert subtrees" in HTML)

When testing (without the negative `tabindex`), I have no problem tabbing onto the link inside the `iframe`. I haven't tested whether screen reader consider it as inert, but Chrome doesn't.